### ISSUE TITLE
[Bug] kubectl grep unusable out of the box

### ISSUE THREAD

**Issue by JJJJJones** [guessi/kubectl-grep#24]
**Describe the bug**
Unless there is a kubeconfig with configuration for a valid cluster set as the current context, all kubectl-grep commands fail.

[Source of the problem](https://github.com/guessi/kubectl-grep/blob/master/pkg/utils/utils.go#L19)
[InitClient() code](https://github.com/guessi/kubectl-grep/blob/master/pkg/client/client.go#L22)

**To Reproduce**
1. Generate an unacceptable or null kubeconfig either by removing all context entries, deleting random lines, corrupting values or outright deleting kubeconfig
2. Try any invocation of `kubectl grep`

**Expected behavior**
`kubectl grep --help` will show help info
`kubectl grep --version` will show the version
`kubectl grep pods "debug"` will fail

**Screenshots**
```shell
sh-4.2# kubectl grep --help
Fatal: Unable to get config
sh-4.2# kubectl grep --version
Fatal: Unable to get config
sh-4.2# kubectl grep pods "debug"
Fatal: Unable to get config
sh-4.2# kubectl config view
apiVersion: v1
clusters: []
contexts: []
current-context: ""
kind: Config
preferences: {}
users: []
sh-4.2# kubectl-grep --version
Fatal: Unable to get config
sh-4.2# kubectl-grep --help
Fatal: Unable to get config
sh-4.2#
```

**Environment (please complete the following information):**
 - Operation System: [e.g. Ubuntu, CentOS, macOS]
MacOS 10.15.3, CentOS 7/8, Docker 2.2.0.4
 - Kubernetes Version [e.g. v1.10.5, v1.11.7, v1.13.4]
v1.16.6
 - Kubectl Version [e.g. v1.10.5, v1.11.7, v1.13.4]
v1.16.6

**Additional context**
Its very disruptive for automated utilities that login to a cluster must be done to verify the version installed.

**Comment by guessi** [comment]
@JJJJJones thanks for the bug report, the issue have been fixed with [kubectl-grep v1.2.5](https://github.com/guessi/kubectl-grep/releases), please try to update the binary on your side.

**Comment by JJJJJones** [comment]
Thanks you @guessi!
